selfdrive/common/tests/test_util.cc

```diff
@@ -0,0 +1,98 @@
+
+#include <dirent.h>
+#include <sys/types.h>
+
+#include <algorithm>
+#include <climits>
+#include <random>
+#include <string>
+
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
+#include "selfdrive/common/util.h"
+
+std::string random_string(size_t length) {
```

Comment:
Do we really need random data? Why not a function that takes in a length, and an upper and lower bound on the byte values? `random_bytes` can be between 0 and 255, `random_string` is between 32 and 126.

But why this distinction between ascii and non ascii? Nothing in the code that we test does anything different for ascii.